### ISSUE TITLE
No SDL.h file?

### ISSUE THREAD

**Issue by andixum** [lantus/cannonball-nx#6]
I installed sdl2 and devkitpro and tried to build this. but keep getting an error that SDL.h doesnt exist, do I have to create it somehow?

**Comment by lantus** [comment]
did you install SDL?

ie 'pacman -S switch-sdl2'

**Comment by andixum** [comment]
yeah i did that, reinstalled just in case and still doesn't work.
